### ISSUE TITLE
Ability to support multipart/jpeg (streaming)

### ISSUE THREAD

**Issue by pliablepixels** [dougmoscrop/angular-img-http-src#12]
Hi, thanks for your directive.
I am considering using this in a system that uses `<img src=xxxx>` to stream multipart JPEG images from cameras. It is an open source home security system. The problem with multipart JPEG is the `$http.get` will never terminate. The standard browser `<img src>` takes care of this automatically. Is it possible to support this? thanks.

**Comment by dougmoscrop** [comment]
Interesting - the browser like auto updates the image every time a new part is received?

Supporting it here would probably depend on what's available from Angular?

**Comment by pliablepixels** [comment]
Yes. The image keeps getting updated after each multipart segment is received.
